ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -376,6 +368,7 @@ - (void)testShouldHandleFileOpEvent {
             &testRootHandle,
             &pid));
 
+    org_vfsforgit_PrjFSProviderUserClient userClient;
```

Comment:
I realise the old code did this too, but the `#define`d name, `PrjFSProviderUserClient` is generally preferable in code - that's the whole point of the macro, if the reverse DNS prefix changes, you don't have to update all of the code.

Might be worth doing a search for the use of the explicitly-prefixed name, I notice quite a few instances of it being used in this file.